instrumentation/spring/spring-boot-actuator-autoconfigure-2.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/actuator/v2_0/ActuatorTest.java

```diff
@@ -57,9 +60,19 @@ void shouldInjectOtelMeterRegistry() {
                                                         "value"))))));
 
     MeterRegistry meterRegistry = context.getBean(MeterRegistry.class);
-    assertThat(meterRegistry).isNotNull().isInstanceOf(CompositeMeterRegistry.class);
-    assertThat(((CompositeMeterRegistry) meterRegistry).getRegistries())
-        .anyMatch(r -> r.getClass().getSimpleName().equals("OpenTelemetryMeterRegistry"))
-        .anyMatch(r -> r.getClass().getSimpleName().equals("SimpleMeterRegistry"));
+    assertThat(meterRegistry).isInstanceOf(CompositeMeterRegistry.class);
+    AbstractCollectionAssert<
+            ?, Collection<? extends MeterRegistry>, MeterRegistry, ObjectAssert<MeterRegistry>>
+        match =
+            assertThat(((CompositeMeterRegistry) meterRegistry).getRegistries())
+                .anyMatch(r -> r.getClass().getSimpleName().equals("OpenTelemetryMeterRegistry"))
+                .anyMatch(r -> r.getClass().getSimpleName().equals("SimpleMeterRegistry"));
```

Comment:
Considering that the issue is about ordering consider whether it would make sense to test that these registries appear in the right order.